spacy/training/loggers.py

```diff
@@ -137,3 +137,77 @@ def finalize() -> None:
         return log_step, finalize
 
     return setup_logger
+
+
+@registry.loggers("spacy.CometLogger.v1")
+def comet_logger(project_name: str, remove_config_values: List[str] = []):
+    try:
+        import comet_ml
+    except ImportError:
+        print("You need to install `comet_ml` to use the CometLogger: see: https://comet.ml/docs/python-sdk/spacy/")
+
+    console = console_logger(progress_bar=False)
+
+    def setup_logger(
+        nlp: "Language", stdout: IO = sys.stdout, stderr: IO = sys.stderr
+    ) -> Tuple[Callable[[Dict[str, Any]], None], Callable[[], None]]:
+
+        try:
+            experiment = comet_ml.Experiment(project_name=project_name)
+        except Exception:
+            experiment = None
+            print("You need to configure Comet, and run appropriately: see: https://comet.ml/docs/python-sdk/spacy/")
+
+        # Get the config, removing items if necessary:
+        config = nlp.config.interpolate()
+        config_dot = util.dict_to_dot(config)
+        for field in remove_config_values:
+            del config_dot[field]
+        config = util.dot_to_dict(config_dot)
```

Comment:
I think this code is identical to what the W&B logger is using - so perhaps it makes sense to move this to a helper function, so that all loggers easily have access to that functionality and we avoid duplication?

Reply:
Moved to utils